tasks/recover/add_domain.yml

```diff
@@ -1,38 +1,43 @@
 - ovirt_hosts_facts:
       pattern: "status=up and datacenter={{ storage['dr_' + dr_target_host + '_dc_name'] }}"
       auth: "{{ ovirt_auth }}"
+- block:
+  - name: "Check for available hosts"
+    fail: msg="No hosts available"
+    when: ovirt_hosts.0 is undefined
```

Comment:
I would use `when len(ovirt_hosts) == 0`

Reply:
I get "The conditional check 'len(ovirt_hosts) == 0' failed. The error was: error while evaluating conditional (len(ovirt_hosts) == 0): 'len' is undefined"

I can use ovirt_hosts | length == 0 instead, is that OK?

Reply:
Ah, right. Sorry.